MAUI/StepProgressBar/migration.md

```diff
@@ -0,0 +1,425 @@
+
+The following code example, explains how to initialize the properties of the `Xamarin SfStepProgressBar` and `.NET MAUI SfStepProgressBar` class.
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.XForms.ProgressBar;assembly=Syncfusion.SfProgressBar.XForms">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.ProgressBar;
+...
+
+SfStepProgressBar sfStepProgressBar = new SfStepProgressBar ();
+this.Content = sfStepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+xmlns:progressBar="clr-namespace:Syncfusion.Maui.ProgressBar;assembly=Syncfusion.Maui.ProgressBar">
+
+    <progressBar:SfStepProgressBar />
+
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.ProgressBar;
+...
+
+SfStepProgressBar stepProgressBar = new SfStepProgressBar();
+this.Content = stepProgressBar ;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+<table>
+<tr>
+<th>Xamarin SfStepProgressBar</th>
+<th>.NET MAUI SfStepProgressBar</th>
+<th>Description</th>
+</tr>
+<tr>
+<td>{{'[CompletedStepStyle]()'| markdownify }}</td>
+<td>{{'[CompletedStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the completed step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[InProgressStepStyle]()'| markdownify }}</td>
+<td>{{'[InProgressStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the in progress step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[NotStartedStepStyle]()'| markdownify }}</td>
+<td>{{'[NotStartedStepSettings]()'| markdownify }}</td>
+<td>Gets or sets the not started step style for the step progressbar.</td>
+</tr>
+<tr>
+<td>{{'[Orientation]()'| markdownify }}</td>
+<td>{{'[Orientation]()'| markdownify }}</td>
+<td>Gets or sets the orientation.</td>
+</tr>
+<tr>
+<td>{{'[ProgressAnimationDuration]()'| markdownify }}</td>
+<td>{{'[ProgressAnimationDuration]()'| markdownify }}</td>
+<td>Gets or sets the progress animation duration.</td>
+</tr>
+<tr>
+<td>{{'[TitleAlignment]()'| markdownify }}</td>
+<td>{{'[LabelPosition]()'| markdownify }}</td>
+<td> Gets or sets the label position or tile alignment.</td>
+</tr>
+<tr>
+<td>{{'[TitleSpace]()'| markdownify }}</td>
+<td>{{'[LabelSpacing]()'| markdownify }}</td>
+<td>Gets or sets the label spacing.</td>
+</tr>
+<tr>
+<td>{{'[LayoutOption]()'| markdownify }}</td>
+<td>Nil</td>
+<td>Not Supported.</td>
+</tr>
+<tr>
+<td>{{'[MarkerSize]()'| markdownify }}</td>
+<td>{{'[StepSize]()'| markdownify }}</td>
+<td>Gets or sets the size of the step.</td>
+</tr>
+<tr>
+<td>{{'[MarkerStrokeWidth]()'| markdownify }}</td>
+<td>{{'[StepStrokeWidth]()'| markdownify }}</td>
+<td>Gets or sets the stroke width of the step.</td>
+</tr>
+</table>
+
+### StepSettings
```

Comment:
<img width="1018" alt="Screenshot 2024-03-12 at 6 23 26 PM" src="https://github.com/syncfusion-content/maui-docs/assets/46158936/dcbab821-7379-4cff-9ccd-6e21a2b26e3c">

dont mention as text style- explain it is used for Primary text

Reply:
I have changed the description as below
Gets or sets the font attributes of the text style which is used to customize the primary and secondary text.